config/config.yaml

```diff
@@ -30,6 +30,7 @@ service:
     SECRET_KEY: aardvark
     EMAIL_TOKEN_SALT: aardvark
     PARTY_SCHEMA: ras_party/schemas/party_schema.json
+    PUBLIC_EMAIL_VERIFICATION_URL: aardvark                   # Do not add trailing slash e.g. http://x.y/parent/child/ BAD http://x.y/parent/child GOOD
```

Comment:
It looks like the frontstage dependency is redundant - can we remove it from the config file.

Reply:
I'll add as a TODO and put in release notes as i need to merge this for 5.  don't want to break anything.